scripts/test_wheel.sh

```diff
@@ -94,7 +103,40 @@ fi
 
 test -d deps/env || USE_MINIMAL="$USE_MINIMAL" ./scripts/install_python_toolchain.sh
 source deps/env/bin/activate
-pip install target/turicreate-*.whl
+
+bdist_wheels=($(ls target/turicreate-*.whl))
+if [[ ${#bdist_wheels[@]} -eq 1 ]]; then
+  wheel_to_install=${bdist_wheels[0]}
+  if [[ "$USE_MINIMAL" -eq 1 ]]; then
+    if [[ ! "$wheel_to_install" =~ .*"+minimal".* ]]; then
+      echo "minimal pkg is not found. Only found $wheel_to_install"
+      exit 1
+    fi
+  else
+    if [[ "$wheel_to_install" =~ .*"+minimal".* ]]; then
+      echo "full pkg is not found. Only found $wheel_to_install"
+      exit 1
+    fi
+  fi
+
+elif [[ ${#bdist_wheels[@]} -eq 2 ]]; then
+  wheel_to_install=${bdist_wheels[0]}
+  if [[ $USE_MINIMAL -eq 1 ]]; then
+    if [[ ! "$wheel_to_install" =~ .*"+minimal".* ]]; then
+      wheel_to_install="${bdist_wheels[1]}"
+    fi
+  else
+    if [[ "$wheel_to_install" =~ .*"+minimal".* ]]; then
+      wheel_to_install="${bdist_wheels[1]}"
+    fi
+  fi
+
+else
+  echo "wrong number of wheel: ${#bdist_wheels[@]}. At most 2 pkgs are needed."
```

Comment:
`wheel` -> `wheels`

I don't understand what you mean by "pkgs". Packages? It probably makes more sense to just say wheels.

Reply:
wheel is a package format. Therefore wheel file is a package. I can change it to wheel.